tests/utils/test_docstring_utils.py

```diff
@@ -46,9 +46,9 @@ def multiple_params(_p1, _p2):
 
     expected_doc = """
         :param p1:
-            param1 doc
+                param1 doc
```

Comment:
This seems to be wrong?

Reply:
Can you elaborate? The goal is to indent by 8 spaces instead of 4.